icd/api/include/pipeline_compiler.h

```diff
@@ -115,8 +115,11 @@ class PipelineCompiler
         Util::MetroHash::Hash*              pCacheId);
 
     VkResult SetPipelineCreationFeedbackInfo(
-        const VkPipelineCreationFeedbackCreateInfoEXT* pPipelineCreationFeadbackCreateInfo,
-        const PipelineCreationFeedback*                pPipelineFeedback);
+        const VkPipelineCreationFeedbackCreateInfoEXT*
+            pPipelineCreationFeadbackCreateInfo,
+        const VkGraphicsPipelineCreateInfo* pCreateInfo,
+        const PipelineCreationFeedback* pPipelineFeedback,
+        const PipelineCreationFeedback* pStageFeedback);
```

Comment:
nit: these argument names should be aligned

Reply:
done